packages/web-components/fast-element/docs/guide/observables-and-state.md

```diff
@@ -22,22 +22,93 @@ The `@attr` decorator can only be used in a `FASTElement` but the `@observable`
+Properties with only a getter, that function as a computed property over other observables, should not be decorated with `@attr` or `@observable`. However, they may need to be decorated with `@volatile`, depending on the internal logic.
 :::
 
-### Features
+## Observable Features
 
-#### Access tracking
+### Access tracking
 
-When decorated properties are accessed during template rendering, they are tracked, allowing the engine to deeply understand the relationship between your model and view.
+When `@attr` and `@observable` decorated properties are accessed during template rendering, they are tracked, allowing the engine to deeply understand the relationship between your model and view. These decorators serves to meta-program the property for you, injecting code to enable the observation system. However, if you do not like this approach, for `@observable`, you can always implement notification manually. Here's what that would look like:
 
-#### Internal observation
+**Example: Manual Observer Implementation**
 
-On the class where the attr/observable is defined, you can optionally implement a *propertyName*Changed method to easily respond to changes in your own state.
+```ts
+import { Observable } from '@microsoft/fast-element';
+
+export class Person {
+  private _firstName: string;
+  private _lastName: string;
+
+  get firstName() {
+    Observable.track(this, 'firstName');
+    return this._firstName;
+  }
+
+  set firstName(value: string) {
+    this._firstName = value;
+    Observable.notify(this, 'firstName');
+  }
+
+  get lastName() {
+    Observable.track(this, 'lastName');
+    return this._lastName;
+  }
+
+  set lastName(value: string) {
+    this._lastName = value;
+    Observable.notify(this, 'lastName');
+  }
+
+  get fullName() {
+    return `${this.firstName} ${this.lastName}`;
+  }
+}
+```
+
+Notice that the `fullName` property does not need any special code, since it's computing based on properties that are already observable. There is one special exception to this: if you have a computed property with branching code paths, such as ternary operators, if/else conditions, etc, then you must tell the observation system that your computed property has *volatile* dependencies. In other words, which properties need to be observed may change from invocation to invocation based on which code path executes. 
+
+Here's how you would do that with a decorator:
+
+```ts
+import { observable, volatile } from '@microsoft/fast-element';
+
+export class MyClass {
+  @observable someBoolean = false;
+  @observable valueA = 0;
+  @observable valueB = 42;
+
+  @volatile
+  get computedValue() {
+    return this.someBoolean ? this.valueA : this.valueB;
+  }
+}
+```
+
+Here's how you would do it without a decorator:
+
+```ts
+import { Observable } from '@microsoft/fast-element';
+
+export class MyClass {
+  // elided
```

Comment:
elided?

Reply:
I just didn't want to repeat all the property definitions again from the example above. I can say "properties elided" if that's clearer.

Reply:
I went ahead and expanded the code out to help clarify the scenario.